Gulpfile.js

```diff
@@ -117,12 +114,13 @@ gulp.task("test", done =>
             return;
         }
 
-        execTask(_AVA_ +  " " + lMatchingFiles.join(" "), done);
+        lArgs = lMatchingFiles;
     }
     else
     {
-        execTask(getAvaCommand(TestFolder) + getAvaArgs("match") + getAvaArgs("serial"), done);
+        lArgs = [...getAllTestFiles(TestFolder), "--serial", "--match"];
     }
+    spawnTask(_AVA_, done, lArgs);
```

Comment:
I like a new line after the end of a logical block (`if/else`)
```suggestion

    spawnTask(_AVA_, done, lArgs);
```

Reply:
done,